.github/workflows/integration.yml

```diff
@@ -106,12 +106,10 @@ jobs:
       - name: Run your tests with reporting
         if: ${{ env.USE_CURRENTS == 'true' }}
         working-directory: tests/acceptance
-        run: |-
-          npx pwc \
-            --ci-build-id ${CURRENTS_BUILD_ID} \
```

Comment:
CURRENTS_BUILD_ID was undefined, this messed the call up

Reply:
Previously, this was [constructed](https://github.com/shopware/shopware/pull/5895/files) based on the environment